server/migrations/20230411235447_activity_char_update.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema.alterTable('entries', (table) => {
+    table.string('activity', 10000).alter();
+  });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = async function(knex) {
+  await knex.schema.alterTable('entries', (table) => {
+    table.dropColumn('activity');
+  });
+  await knex.schema.alterTable('entries', (table) => {
+    table.string('activity');
+  })
+};;
```

Comment:
BIG BACKEND BRAIN STUFF!! You are a genius.

Reply:
Curious that it let us do alter table here, but it wasn't letting us use it the other day? I'm looking forward to understanding knex a little better as we go :)

Reply:
same, would love to chat about this